wallet/lib/pages/purchase_item/purchase_item_screen.dart

```diff
@@ -744,7 +744,7 @@ class _OwnerBottomDrawerState extends State<OwnerBottomDrawer> {
         createdBy: viewModel.nft.creator,
         currency: viewModel.nft.ibcCoins.getAbbrev(),
         soldBy: viewModel.nft.owner.isEmpty ? viewModel.nft.creator : viewModel.nft.owner,
-        transactionTime: "${formatter.format(DateTime.now().toUtc())} UTC",
+        transactionTime: formatter.format(DateTime.now()),
```

Comment:
@faizhashmiRnS I thought we agreed to use UTC time. This is using the local time now. And with the previous code it wasn't converting to UTC time anyways.

Reply:
@mustafapylons agreed.

Reply:
@mustafapylons  Oh Alright. Let me look into it. Thanks

Reply:
@faizhashmiRnS  Sorry but i am just seeing it now. If you see the code the dateTime.now means that we are not getting it or using it from the chain.

Reply:
> @faizhashmiRnS Sorry but i am just seeing it now. If you see the code the dateTime.now means that we are not getting it or using it from the chain.

Hi, I just had a talk with Sir Faisal. Backend team will work on it.